package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "express-gateway",
-  "version": "1.12.1",
+  "version": "1.12.2",
```

Comment:
Do not advance the `package.json` version, we take care of it when it is time to release it.

Reply:
Noted